src/Maker/MakeEntity.php

```diff
@@ -48,12 +48,27 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
 
     public function getParameters(InputInterface $input): array
     {
-        $entityClassName = Str::asClassName($input->getArgument('entity-class'));
+        $classname = $input->getArgument('entity-class');
+        
+        $parts = explode('\\',$classname);
+        $short_classname = $parts[count($parts)-1] ;
+        $namespace = implode('\\',array_slice($parts,0,-1));
+        $classname = $short_classname ;
+
+        $path = str_replace('\\','/',$namespace);
+
+        if ($namespace) {
+            $path = $path.'/' ;
+        }
```

Comment:
This should somehow be extracted into an external class, as we'll probably want to re-use this other places too.